### ISSUE TITLE
[Bug] focus on first item when entering a hub (classic mode)

### ISSUE THREAD

**Issue by SJ1O** [jurialmunkey/skin.arctic.fuse#266]
### Skin section

Home Widgets

### Feature description

It would be nice if when you click a side menu option it would automatically focus on the first item within the selected hub. For example you click movies it focuses on the first item within the movies hub. Update. I have just been playing about and noticed this behaviour only happens when the hub is set up for classic mode (widgets only) it functions as expected in hubs that have categories.

### Why is this feature needed? Please justify and explain.

I can think of two reasons. 
1. If you enter a hub it automatically focuses on the home menu icon and could be confusing.
2. If you enter a hub movies for example Then enter the Home menu again it automatically focuses on the movies hub icon.

I believe the previous functionality to be better when it always focused on the entered hub.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by SJ1O** [comment]
[

https://github.com/jurialmunkey/skin.arctic.fuse/assets/121204283/fb188d13-8af1-47af-bbe4-42a76de384fc


https://github.com/jurialmunkey/skin.arctic.fuse/assets/121204283/63d384b3-6c91-471b-9f36-303c0d23d542

](url)

**Comment by jurialmunkey** [comment]
Is this not an issue anymore?

**Comment by jurialmunkey** [comment]
Ah I see you've collated the issues in a new thread #273

**Comment by SJ1O** [comment]
> Ah I see you've collated the issues in a new thread #273

Yes I thought it would be best I explained abit better and pinpointed and labeled the issues rather then adding and explaining in the comments. Figured it would be cleaner for you to look at and less confusing as I know I don't always make to much sense just trying to make your life a tad easier.

Thanks for looking into this it's much appreciated. I have finally started to use the categories in one of my hubs and really enjoyed that feature. I created a documentary hub and created categories for history, crime, nature, survival it's so much easier to find what I want. I might even end up doing something similar in other hubs  just gotta think it through. I did have a read of the widget ideas page another user created and your setup sounds interesting but I see your also using classic mode so not sure what route I'm going to take but It would be nice to see some decent recommendations for movies and shows so I might end up creating categories for that. I don't no yet but I do enjoy playing about with it all. Just annoyes others in the house when I constantly change things :)
